mw-webapp/src/component/header/Header.cy.tsx

```diff
@@ -0,0 +1,24 @@
+
+import {App} from "src/App";
+import {getDataCy} from "src/utils/cyTesting/getDataCy";
+
+const HEADER_CY = "header";
+const LOGO_ALT_TEXT = "Master's way";
+
+describe("Header component", () => {
+
+  beforeEach(() => {
+    cy.mount(
+      <App />,
+    );
+  });
+
+  it("Alt text visible if source broken", () => {
+    cy.get(getDataCy(HEADER_CY)).find("img").should("have.attr", "alt", LOGO_ALT_TEXT);
+  });
+
+  it("Image visible if source exists", () => {
+    cy.get(getDataCy(HEADER_CY)).find("img").should("be.visible");
```

Comment:
!

Reply:
I mount <App /> because Header use useGlobalContext and routing? we have mistakes with that
![image](https://github.com/tritonJS826/masters-way/assets/112805668/62e92311-c988-4f9f-9f9c-d507c8f71d47)
I try to use wrapper with useGlobalContext , but we have another mistake with routing, but i don't know how to do wrapper with our routings

Reply:
answered in video)

Reply:
fixed, add wrapper